shop/src/app/mocks/products.mocks.ts

```diff
@@ -1,36 +1,39 @@
-import { ProductDetails } from "../modules/shared/types/product-details.types";
-import { ShoppingCartProduct } from "../modules/shared/types/shopping-cart-product.types";
+// import { ProductDetails } from "../modules/shared/types/mock-types/product-details.types";
+// import { Product } from "../modules/shared/types/product.types";
 
-export function getProductDetailsMock(name:string = "product1"):ProductDetails{
-    return {
-        name : name,
-        category: "string",
-        price: 10,
-        description: "string"
-    };
-}
+// export function getProductDetailsMock(name:string = "product0"):ProductDetails{
+//     return {
+//         name : name,
+//         category: "string",
+//         price: 10,
+//         description: "string"
+//     };
+// }
```

Comment:
Do you need this commented code? Same for the ones below

Reply:
Deleted the file